mono/metadata/mono-perfcounters.c

```diff
@@ -1600,113 +1665,125 @@ mono_perfcounter_instance_exists (MonoString *instance, MonoString *category, Mo
 	return FALSE;
 }
 
-/* this is an icall */
-MonoArray*
-mono_perfcounter_category_names (MonoString *machine)
+static void
+mono_new_string_into_array (
```

Comment:
Don't prefix by `mono` if it's a static function, it's just confusing as we might expect it to be a non-static function that is accessible from elsewhere.

Reply:
ok